CGCharter-1727386911.html

```diff
@@ -62,28 +62,59 @@ <h2 id="goals">
         Incubate new proposals which build on or complement the Social Web WG recommendations.
       </li>
     </ul>
-  
+
     <h2 id="scope-of-work">
       Scope of Work
     </h2>
-    <p class="remove">
-      {TBD: Describe topics that are in scope. For specifications that the CLA
-      patent section applies to, it is helpful to describe the scope in a way
-      that it is clear what types of technologies will be defined in
-      specifications, as opposed to adoption by reference or underlying
-      technology not defined in the proposed spec. Key use cases are often
-      helpful in describing scope. If no specifications will be defined in the
-      group that the CLA patent section applies to, the charter should clearly
-      state that. A clear scope is particularly important where patent
-      licensing obligations may apply.}
+    <p>
+      The group will maintain errata and corrected drafts, consisting of the published document with errata applied, for the following recommendations from the Social Web Working Group:
+    </p>
+    <ul>
+      <li>ActivityPub</li>
+      <li>Activity Streams 2.0 Core</li>
+      <li>Activity Vocabulary</li>
+      <li>Webmention</li>
+      <li>Linked Data Notifications</li>
+      <li>Micropub</li>
+      <li>WebSub</li>
+    </ul>
+    <p>
+      It will maintain errata and corrected drafts, consisting of the published document with errata applied, for the following Notes from the Social Web Working Group:
+    </p>
+    <ul>
+      <li>JF2</li>
+      <li>Post Type Discovery</li>
+      <li>Social Web Protocols</li>
+      <li>IndieAuth</li>
+    </ul>
+    <p>
+      The group will maintain the Activity Streams 2.0 JSON-LD context document and add new extension terms as they become widely implemented.
+    </p>
+    <p>
+      The group will maintain the Activity Streams 2.0 OWL ontology.
```

Comment:
```suggestion
      The group may publish the Activity Streams 2.0 ontology in machine-readable formats like OWL over time, and will make clear in these publications the degree of maintenance to expect from each over time.
```

I'm hesitating to list everything we WILL maintain, since this particular item was never really finished or ratified as a consensus deliverable; i definitely encourage volunteers and hope we can help find maintainers long-term, though.

Reply:
More generally, it may be good to introduce the general concept of machine-readable Reports or other artefacts that are _downstream of_ maintained items, like schemata, beyond just the one ontology for RDF normalization. Mental note to bring it up in today's meeting and add an issue if others agree.